### ISSUE TITLE
ciclop 3d scanner reprap evolution board

### ISSUE THREAD

**Issue by Turlo77** [bqlabs/horus#104]
Hi I have a ciclop 3d scanner but cant seem to get it to work.
I am using Horus on a Mac running OSX 10.12.3 (Siera), but most the support doesn't correspond to the board I have installed in the scanner. The board is an evolution board.
![evolution board1](https://cloud.githubusercontent.com/assets/25770798/22931299/05083bf2-f2bc-11e6-8391-e5af06b8fbc0.jpg)
![evolution board2](https://cloud.githubusercontent.com/assets/25770798/22931298/050740ee-f2bc-11e6-8415-72bd67350a0a.jpg)
The error message is "The board has the wrong firmware or an invalid Baud Rate. Please select your board and press 'update firmware'" when I do this I get the error "Probably you have selected the wrong board. Select another board".
Anyone else having this issue or can help please?

**Comment by machogg** [comment]
I got a similar board in a HE3D Cyclops scanner and had the same result with my Mac.  Replaced it with an Uno and BQ shield and was able to make a connection.  Still have a problem with getting the motor to run smoothly so this may not be a total solution (still cannot scan anything because it will not calibrate )

**Comment by blackjec69** [comment]
Some version of Horus use different calibration method (old version - "chess-board" placed to far side of platform relative camera, new version - to near). See pictograms into instruction inside Horus during calibration.

**Comment by nosleduc** [comment]
I've bought HE3D ciclop 3D and I had the same issue. I'm using OSX Sierra.
I've changed in the preferences window AVT board to Arduino Uno and click upload firmware.
After restarting, the system worked

**Comment by Neovision5** [comment]
I Have the same HE3D ciclop scanner with the evolution board stock attempting to utilize with OSX Sierra 10.12.4.  My issue is the USB not connected symbol that is being displayed, throughout the Horus and the setup wizard. I looked into the logitech c70 720p cam it uses, tested the cam with Logitech Camera Settings, where the cam did connect and work. Horus on the other hand isn't recognizing it. Anyone know how to fix this? Any light shed would be much appreciated. 

![image](https://cloud.githubusercontent.com/assets/28714711/26074901/781595c0-3981-11e7-95fa-2aec7d95c1f7.png)

**Comment by nosleduc** [comment]
Did you see two camera ids in the preferences (0 and 1)?

**Comment by Neovision5** [comment]
Yes, 0 is the c270 and 1 is the macbook camera. When I set it to 0 and attempt to connect with the Horus Wizard, the green light on the webcam turns on only for a couple seconds and then shuts off (the whole time showing the USB disconnect symbol where the video should be).

**Comment by nosleduc** [comment]
I replugged my cliclop and I had similar issues. Instead of using the wizard, I opened the preferences and set the following parameters. After changing the parameters, I quit and restart the application.

CameraID: 1
Serial Name: /dev/cu.wchusbserial1410
Baud rate: 115200
AVB Board: Arduino Uno
Binary file: Default
